.github/workflows/pull-request.yml

```
@@ -253,6 +253,45 @@
       - name: Validate Documents
         run: graphql-inspector validate './src/GraphQl/**/*.ts' './talawa-api/schema.graphql'
 
+  Docker-Start-Check:
+    name: Check if Talawa Admin app starts in Docker
+    runs-on: ubuntu-latest
+    needs: [Code-Quality-Checks, Test-Application]
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v4
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)